bridges/snowbridge/primitives/router/src/inbound/mod.rs

```diff
@@ -278,8 +278,8 @@ where
 						xcm: vec![
 							// Buy execution on target.
 							BuyExecution { fees: dest_para_fee_asset, weight_limit: Unlimited },
-							// Deposit asset to beneficiary.
-							DepositAsset { assets: Definite(asset.into()), beneficiary },
+							// Deposit both asset and fees to beneficiary.
+							DepositAsset { assets: Wild(AllCounted(2u32)), beneficiary },
```

Comment:
Why `AllCounted` instead of `All`?

Reply:
https://github.com/Snowfork/polkadot-sdk/pull/137/commits/f28b8db3c00be0945f127ac840333c713f1f2462